bins/more/sim.fzb

```diff
@@ -269,35 +192,7 @@
             </views>
         </instance>
         <!--
-        <instance modelIndex="38" moduleIdRef="4070_4_x_2_input_XOR_gate_multipart">
-            <views>
-                <iconView layer="icon">
-                    <geometry x="-1" y="-1" z="-1"/>
-                </iconView>
-            </views>
-        </instance>
-        <instance modelIndex="38" moduleIdRef="4071_4_x_2_input_OR_gate_multipart">
-            <views>
-                <iconView layer="icon">
-                    <geometry x="-1" y="-1" z="-1"/>
-                </iconView>
-            </views>
-        </instance>
-        <instance modelIndex="38" moduleIdRef="4077_4_x_2_input_XNOR_gate_multipart">
-            <views>
-                <iconView layer="icon">
-                    <geometry x="-1" y="-1" z="-1"/>
-                </iconView>
-            </views>
-        </instance>
-        <instance modelIndex="38" moduleIdRef="4081_4_x_2_input_AND_gate_multipart">
-            <views>
-                <iconView layer="icon">
-                    <geometry x="-1" y="-1" z="-1"/>
-                </iconView>
-            </views>
-        </instance>
-        -->
+        <instance modelIndex="38" moduleIdRef="4070_4_x_2_input_XOR_gate_multipart"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance modelIndex="38" moduleIdRef="4071_4_x_2_input_OR_gate_multipart"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance modelIndex="38" moduleIdRef="4077_4_x_2_input_XNOR_gate_multipart"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance><instance modelIndex="38" moduleIdRef="4081_4_x_2_input_AND_gate_multipart"><views><iconView layer="icon"><geometry x="-1" y="-1" z="-1"/></iconView></views></instance>-->
```

Comment:
Can it be removed?

Reply:
Same as in the  previous comment: These lines are commented as these parts do not have SPICE fields currently, but they could be added more or less easily. It is a kind of TODO list for me, but we can remove them if you prefer it.